src/application/use-cases/product/CreateProductUseCase.ts

```
@@ -0,0 +1,58 @@
+export interface CreateProductUseCase {
+    category: string;
+    productName: string;
+    description: string;
+    price: number;
+    discount?: number;
+    addSize?: SizeOption;
+    image?: Express.Multer.File;
+    restaurantOwnerId: string;
+}
+
+export class CreateProductUseCase {
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)